src/layoutcomponent/Footer.jsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import "../styles/Footer.css";
+
+const Footer = () => {
+  return (
+    <div>
+      {" "}
+      <footer>
+        <div class="footer-drinks-break-main">
```

Comment:
Change `class` to `className` everywhere

Reply:
sure in next pr i will update this